controllers/apply/lib/field-types/field.js

```diff
@@ -30,7 +29,11 @@ class Field {
 
         this.type = props.type ? props.type : this.getType();
 
-        this.attributes = defaults(this.defaultAttributes(), props.attributes);
+        this.attributes = Object.assign(
```

Comment:
`defaults` has this behaviour: "Once a property is set, additional values of the same property are ignored". This isn't what we want so switch to `Object.assign`